source/core/TypeAndDataManager.cpp

```diff
@@ -1973,12 +1973,12 @@ RefNumValType::RefNumValType(TypeManagerRef typeManager, TypeRef type)
     _isFlat = true;
     _hasCustomDefault = true;
     _topAQSize = sizeof(RefNumVal);
+    _aqAlignment = sizeof(RefNumVal) > 4 ? 8 : 4;
```

Comment:
Added comment to explain, and use alignof(RefNumVal) instead of this since we can use C++11 now.
(Without this, it used the wrappedType's alignment, which is wrong, since we don't store that directly, only via cookie.)